src/utils.ts

```diff
@@ -86,48 +65,32 @@ export const getBrowserType = (browser?: BrowserType): BrowserType => {
   return browser || CHROMIUM
 }
 
-export const readPackage = async (): Promise<
-  Packages | typeof IMPORT_KIND_PLAYWRIGHT
-> => {
-  const packagePath = 'package.json'
-  const absConfigPath = path.resolve(process.cwd(), packagePath)
-  const packageConfig = await require(absConfigPath)
-  // for handling the local tests
-  if (packageConfig.name === 'jest-playwright-preset') {
-    return IMPORT_KIND_PLAYWRIGHT
+export const getPlaywrightInstance = (browserName: BrowserType): Playwright => {
```

Comment:
Very nice! a lot more simplified by that (try catch) so we don't need the dependencies and devDependencies anymore!

But I would simplify it even more, for a better readability, what do you think about that?

```ts
export const getPlaywrightInstance = (browserName: BrowserType): Playwright => {
  try {
    const pw = require(`${IMPORT_KIND_PLAYWRIGHT}-${browserName}`)
    return {
      name: browserName,
      instance: pw[browserName],
      devices: pw['devices'],
    }
  } catch (e) {
    try {
      const pw = require(IMPORT_KIND_PLAYWRIGHT)
      return {
        name: IMPORT_KIND_PLAYWRIGHT,
        instance: pw[browserName],
        devices: pw['devices'],
      }
    } catch (e) {
      throw new Error(
        formatError(`Cannot find playwright package to use ${browserName}`),
      )
    }
  }
}
```

Reply:
@mxschmitt that's OK, but it will cause some TS related issues, that a little hard to fix

Reply:
@mmarkelov tried mine, but was not working as expected. Reverted it for now. Approved 👍